src/diffusers/loaders.py

```diff
@@ -2081,27 +2134,40 @@ def unfuse_lora(self, unfuse_unet: bool = True, unfuse_text_encoder: bool = True
         if unfuse_unet:
             self.unet.unfuse_lora()
 
-        def unfuse_text_encoder_lora(text_encoder):
-            for _, attn_module in text_encoder_attn_modules(text_encoder):
-                if isinstance(attn_module.q_proj, PatchedLoraProjection):
-                    attn_module.q_proj._unfuse_lora()
-                    attn_module.k_proj._unfuse_lora()
-                    attn_module.v_proj._unfuse_lora()
-                    attn_module.out_proj._unfuse_lora()
+        if self.use_peft_backend:
+            from peft.tuners.tuner_utils import BaseTunerLayer
+
+            def unfuse_text_encoder_lora(text_encoder):
+                for module in text_encoder.modules():
+                    if isinstance(module, BaseTunerLayer):
+                        module.unmerge()
+
+        else:
+            warnings.warn(
+                "You are using an old version of LoRA backend. This will be deprecated in the next releases in favor of PEFT"
+                " make sure to install the latest PEFT and transformers packages in the future.",
+                FutureWarning,
+            )
```

Comment:
```suggestion
            deprecate
```

Can you please use the deprecate function here? For an example, see: https://github.com/huggingface/diffusers/blob/e312b2302b5445271198ebed8f2fbcd543633f78/src/diffusers/loaders.py#L2121